presto-hive/src/main/java/io/prestosql/plugin/hive/s3/S3FileSystemType.java

```diff
@@ -17,5 +17,5 @@
 {
     PRESTO,
     EMRFS,
-    HADOOP_DEFAULT,
+    NONE,
```

Comment:
https://github.com/prestosql/presto/pull/1397#discussion_r318933739

> I don't like NONE, as we may end up having s3 fs impl.

I mean -- NONE suggest there will be no s3 support, so one would expect s3:// uris to fail. However, it mustn'y be the case.